src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -200,15 +200,20 @@ public Election submitFinalAccessVoteDataRequestElection(Integer electionId) thr
         } 
         Integer userId = dar.getUserId();
         List<LibraryCard> libraryCards = Objects.nonNull(userId) ? libraryCardDAO.findLibraryCardsByUserId(userId) : Collections.emptyList();
-        if (libraryCards.isEmpty()) {
-            throw new NotFoundException("No Library cards exist for the researcher on this DAR");
-        }
         List<Vote> finalVotes = voteDAO.findFinalVotesByElectionId(electionId);
         // The first final vote to be submitted is what determines the approval/denial of the election
         boolean isApproved = finalVotes.stream().
                 filter(Objects::nonNull).
                 filter(v -> Objects.nonNull(v.getVote())).
                 anyMatch(Vote::getVote);
+        //Users cannot submit a DataAccess approval if the researcher does not have a library card
+        //However Chairs can still approve/disapprove RP elections and/or reject DataAccess elections even if the researcher does not have a Library Card
```

Comment:
this, submitFinalAccessVoteDataRequestElection, is always going to be on an access election and not an rp election there is a separate method for those, but good catch on the negative vote

Reply:
Ah, good point, I should only be checking on the negative vote here

Reply:
Wait, I don't think that's true. Both RP and DataAccess votes, when submitted, always go to `api/dataRequest/${requestId}/vote/${vote.voteId}/final`, and that endpoint calls submitFinalAccessVoteDataRequestElection() always. 

I'm outlining the steps for my own reference (and to make sure I'm not off on anything)

Here's the RP submission as chair on `DacVotePanel.js`
<img width="630" alt="Screen Shot 2021-05-25 at 1 47 05 PM" src="https://user-images.githubusercontent.com/6664778/119544519-c9326d00-bd5f-11eb-8517-de90e019cf0b.png">

Here's `submitVote`...
<img width="1041" alt="Screen Shot 2021-05-25 at 1 49 36 PM" src="https://user-images.githubusercontent.com/6664778/119544775-0eef3580-bd60-11eb-8aa4-b0eaaf37d6ac.png">

Since both RP and DataAccess have `type = FINAL`, it ends up calling `updateFinalAccessDarVote`
<img width="911" alt="Screen Shot 2021-05-25 at 1 51 19 PM" src="https://user-images.githubusercontent.com/6664778/119545025-507fe080-bd60-11eb-8491-1e22f0664e0a.png">

The above code ends up here in `DataRequestVoteResource.js`
<img width="1164" alt="Screen Shot 2021-05-25 at 1 43 53 PM" src="https://user-images.githubusercontent.com/6664778/119544092-650fa900-bd5f-11eb-8d83-7f52c5c1b8b1.png">

Reply:
So, on the UI the RP votes do not have FINAL type and so they will never reach that endpoint.  Here is an example of what the votes on the access review page look like for a chairperson on an open election. There are four votes for a chairperson for the access election, in this case 1442, and two votes for a chairperson for the rp election, 1443.
<img width="1181" alt="Screen Shot 2021-05-26 at 7 38 33 AM" src="https://user-images.githubusercontent.com/77015255/119654073-0d704c80-bdf6-11eb-9447-5fc36a3b5725.png">
<img width="1181" alt="Screen Shot 2021-05-26 at 7 38 58 AM" src="https://user-images.githubusercontent.com/77015255/119654101-16611e00-bdf6-11eb-8774-24216e7e74d3.png">